cmd/opm/alpha/convert-template/convert.go

```diff
@@ -0,0 +1,66 @@
+package converttemplate
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/spf13/cobra"
+
+	"github.com/operator-framework/operator-registry/alpha/template/converter"
+	"github.com/operator-framework/operator-registry/cmd/opm/internal/util"
+)
+
+func NewCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "convert-template",
+		Short: "Convert existing FBC to a supported template type",
+	}
+	cmd.AddCommand(
+		newBasicConvertCmd(),
+	)
+	return cmd
+}
+
+func newBasicConvertCmd() *cobra.Command {
+	var (
+		converter converter.Converter
+		output    string
+	)
+	cmd := &cobra.Command{
+		Use:   "basic [<fbc-file> | -]",
+		Args:  cobra.MaximumNArgs(1),
+		Short: "Generate a basic template from existing FBC",
+		Long: `Generate a basic template from existing FBC.
+
+This command outputs a basic catalog template to STDOUT from input FBC.
+If no argument is specified or is '-' input is assumed from STDIN.
+`,
+		RunE: func(c *cobra.Command, args []string) error {
+
+			switch output {
+			case "yaml":
+				fallthrough
+			case "json":
+				converter.OutputFormat = output
```

Comment:
```suggestion
			case "yaml", "json":
				converter.OutputFormat = output
```

Reply:
Reasoning?  When I wrote it I found (for e.g. [here](https://go.dev/wiki/Switch)) no idiomatic argument for either approach.

Reply:
I was gonna say, if this is possible, that's nice =D I can never remember fallthrough - well not exactly true, diarrhea in German is durchfall. So, I have that mnemonic XD

Reply:
IMO using the `fallthrough` directive:
- increases mental complexity for a reader. They have to first understand what the fallthrough directive does and then build the mental model of "If the yaml case is hit it will fallthrough and do whatever the case underneath of it is, which is the json case and therefore will set the OutputFormat"
- `fallthrough` means execute the logic of the case underneath and could result in an easier introduction of bugs. What happens if someone adds an additional case below the `yaml` case that slips through and completely changes the behavior when `yaml` is specified as the output format?

Instead, using the OR syntax for a switch case is:
- Easier to reason about and build a mental model. Simply looking at the case statement tells me that either the value `yaml` or `json` executes the exact same code.
- Less prone to introducing bugs inadvertently as there is a well defined execution path that is not dependent on the ordering of case statements

Reply:
I guess it's a question of subjective contexts here.  I infinitely prefer the explicit fallthrough idiom over for e.g. C's implicit (and problematic) fallthrough.  
And similarly, I prefer to have a single evaluated condition examined per case, based on experience with other languages.  
Like I said... context. 

Not a significant factor for me so I made the changes to unblock the review.

(Also, eww Per. 😉)